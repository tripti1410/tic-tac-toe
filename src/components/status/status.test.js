import React from "react";
import renderer from "react-test-renderer";
import Status from "./status";

it("renders correctly - default state", () => {
  const tree = renderer
    .create(<Status message="" clickHandle={jest.fn()} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
