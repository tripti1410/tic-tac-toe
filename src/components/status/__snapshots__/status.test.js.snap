// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly - default state 1`] = `
<div
  className="status-container"
>
  Result: 
  
</div>
`;

exports[`test renders correctly - default state 1`] = `
<div
  className="status-container">
  Result: 
  
</div>
`;
